pkg/kubelet/status/status_manager.go

```diff
@@ -426,43 +426,46 @@ func (m *manager) syncPod(uid types.UID, status versionedPodStatus) {
 		// If the Pod is deleted the status will be cleared in
 		// RemoveOrphanedStatuses, so we just ignore the update here.
 		return
+	} else if err != nil {
```

Comment:
The `else` is still useful as it provides more information to a developer and the compiler that the two blocks are related.